### ISSUE TITLE
Midpoint Review [My Education Benefits, VA Education Inbox

### ISSUE THREAD

**Issue by alexander-ferzola** [department-of-veterans-affairs/va.gov-team#36497]
# Steps to complete Midpoint Review 

## Before meeting

**VFS Product Manager**
- [ ]  Create this issue and schedule meeting via Calendly (you are responsible for adding the VFS attendees). 
- [ ] Notify the Collaboration Cycle team of this ticket with scheduled date/time in #vfs-platform-support ([see Slack Workflow User guide](https://depo-platform-documentation.scrollhelp.site/support/Getting-help-from-the-Platform-in-Slack.1439138197.html)). 
- [ ]  Add artifacts listed below to the ticket at least 2 days before scheduled meeting. 

**Platform**
- [ ] Add the Platform reviewers to scheduled meeting
- [ ] Add Zoom information to scheduled meeting (if synchronous)
- [ ] Create feedback tickets and link to the Midpoint Review epic (if no feedback, please comment stating that). 

## After meeting
**VFS Product Manager**
- [ ] Respond to feedback tickets.
- [ ] Close this ticket when review is complete and feedback has been addressed.  

**Platform**
- [ ] Update [Platform Collaboration Point Tracker](https://docs.google.com/spreadsheets/d/1OgPyEvUlNF6EnaYMFAXJkV6FKOvZnlPnbOQ2fAJ7W7A/edit#gid=266151061). 

### Has your team met with the Sitewide Content team for content support? 
- [ ] Yes*
- [ ] No

\*If yes, please include any relevant content artifacts.

## Artifacts

Provide links or documents for the following: 

- [ ] Product outline
 
- [ ] Finalized design prototype or mockup
- [ ] Research plan 

 
- [ ] Conversation guide 

### Content artifacts 

- [ ] Github ticket for any relevant static content page and entry point updates for tool or feature (if applicable)
 
### QA artifacts

[Learn more about QA artifacts](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/quality-assurance/qa-artifacts.md#test-plan)

- [ ] Regression test plans (if there isn't already a regression test plan for this feature in TestRaiProduct)
- [ ] Link to test cases/test plan in TestRail (draft OK) 

## Asynchronous review option
Would you like this review to be asynchronous? 

Please refer to the [Midpoint Review guidance](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Midpoint-review.1781039167.html) for the difference between a synchronous meeting and an asynchronous review.
- [ ] Yes
- [ ] No

## Scheduling
- You will need to use Calendly to schedule both synchronous and asynchronous reviews (asynchronous reviews will not have a Zoom link).
- Open the [Calendly midpoint review calendar](https://calendly.com/collaboration-cycle/design-intent-or-midpoint-review)
- Select a date and time and click “Confirm”
- Add your name and email
- Click "Add Guests" and enter the VFS meeting attendees email addresses. 
- Click Schedule Event

## Additional information
Please refer to [Platform Collaboration Cycle](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/index.html) or the [Midpoint Review Touchpoint](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Midpoint-review.1781039167.html) on Platform website for more information about the Collaboration Cycle.

FYI: @shiragoodman , @chaseakirby

**Comment by shiragoodman** [comment]
confirming that the asynch review is scheduled for Friday 2/18. We'll have feedback to you by EOD.

**Comment by claytonzook** [comment]
@alexander-ferzola 
**RE: research**
Have you updated or modified any of the research artifacts? It looks like the versions in Github are all 4-5 months old. 

- Planned testing times in the [Research Plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/e1c6634e17814495f04f18dc2cf55c56a0027e58/products/my-education-benefits/secure-inbox/Secure%20Inbox%20UT%20-%20Research%20Plan%20OCT2021%20(1).docx) are old. Have you been in contact with Shane about updating those for scheduling with Perigean? Regardless, if you are using the same plan, the dates should be updated in the plan document.

- Are all the tasks on your [Conversation Guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/e1c6634e17814495f04f18dc2cf55c56a0027e58/products/my-education-benefits/secure-inbox/Secure%20Inbox%20UT-%20Conversation%20Guide%20OCT2021.docx), still relevant for any changes made to the design?

- It's recommended that links to the prototype or mockups be included at the given Task in the Conversation Guide. Especially if your team needs to make changes on this artifact, I'd recommend you add those in.

**Comment by allison0034** [comment]
Hi @alexander-ferzola can you please link the prototype so we can review? Thank you!

**Comment by shiragoodman** [comment]
FYI if we do not receive the prototype by 5pm ET today, we will have to postpone this review.

**Comment by alexander-ferzola** [comment]
Hi all,
(CC @shiragoodman)

Apologies for the delayed response, was on PTO this week. Couple points to clarify:

1. The links to the research plan and conversation guide are from the original round of Usability Testing for non-MVP Secure Inbox that has already occurred. They were listed for reference but needed no review. We do not have UT yet scheduled for the MVP VA Education Inbox. (@claytonzook)
2. Because there is no scheduled UT, we do not have any prototype link to share. All designs were included in the VA Education collection links (re-attaching here [https://share.goabstract.com/d47c74f7-251c-4e31-86f6-27bbfeaaf9e3](url)) This is where we were hoping to get design feedback, if possible (@allison0034 ). 

Let us know if you have any other questions. Thank you for your support.

**Comment by allison0034** [comment]
@alexander-ferzola do we need access to see that link you attached? I get this...

No results matched your search.
You could search [all of GitHub](https://github.com/search) or try an [advanced search](https://github.com/search/advanced).

**Comment by alexander-ferzola** [comment]
> @alexander-ferzola do we need access to see that link you attached? I get this...
> 
> No results matched your search. You could search [all of GitHub](https://github.com/search) or try an [advanced search](https://github.com/search/advanced).

Updated the link in my original post, should be working now :)

**Comment by shiragoodman** [comment]
@alexander-ferzola oh yea, thank you for the reminder (about the holiday and the asynch review!). We'll just keep the date for tomorrow and if my team provides feedback by EOD, great. If not, we may need til EOD Monday. I don't think anyone on our team has Monday as a holiday, but I'm confirming now.

**Comment by seanasewell** [comment]
@alexander-ferzola Trying to access the designs using the link you provided however looks like the artboards were removed. Is there an updated link you can provide? Thanks!

**Comment by alexander-ferzola** [comment]
> @alexander-ferzola Trying to access the designs using the link you provided however looks like the artboards were removed. Is there an updated link you can provide? Thanks!

Is this the same link you're trying to access? https://share.goabstract.com/d47c74f7-251c-4e31-86f6-27bbfeaaf9e3

**Comment by seanasewell** [comment]
@alexander-ferzola Yes! This is what I was seeing https://d.pr/i/i45l85 Designs weren't appearing. But now I am seeing the designs so we should be good to go. Thanks!

**Comment by claytonzook** [comment]
Research feedback - It's always a good idea to do some iterative testing, especially since scope has changed from original intent. Sounds like that is already in your plan, so I'll look forward to seeing those research artifacts when ready.

**Comment by jimuhh** [comment]
Correction/clarification on my feedback from @mnorthuis 
**DESIRED EXPERIENCE**

- From the static page, we would want them to log in and go straight to the authenticated tool. 
- If they are already logged in and on the site, if they click the link, they should skip the static page and go straight to the authenticated tool.  
- The static pages are really just for uanuthorized users and search engines.

**Comment by alexander-ferzola** [comment]
Thank you all for your review of the VA Education Inbox. We appreciate all of your timely feedback. Our team will review Icebox, answer any outstanding questions and will continue making updates.
